src/bsoncxx/types/bson_value/value.hpp

```diff
@@ -116,11 +116,21 @@ class BSONCXX_API value {
     ///
     value(bsoncxx::document::view v);
 
+    ///
+    /// Constructs a BSON document value.
+    ///
+    value(bsoncxx::document::value v);
```

Comment:
Wondering why these new constructors were needed?  `bsoncxx::document::value` should implicitly convert to a view and allow us to use the view constructors.  Is there something that is requiring two conversions now?

Reply:
Is this a duplicate comment? If so, I posted an answer to the other similar question.

Reply:
Weird!  I saw your other answer, not sure why my comment is here twice!